### ISSUE TITLE
Push button: enable/disable/reset network & bluetooth

### ISSUE THREAD

**Issue by mpvader** [victronenergy/venus#451]
See image for spec of the button; and spec of the two LEDs:

![image](https://user-images.githubusercontent.com/5200296/60966978-cbc97c00-a319-11e9-97b1-bb1cefbc5049.png)

Same can be done with the Venus GX button (right?)

To undo everything a customer can have done to lock him self out, you'd have to:
- (re-)enable local access point
- reset ethernet, wifi, and any other network settings to dhcp; remove gateway etc.
- remove Remote Console password
- enable Remote Console
- reset the bluetooth pin code (future)

The more rigorous option would be to make it reset all settings to factory defaults. For example by deleting all files except whats in `/data/venus` & `/conf/vrm_auth_token.txt`, for example, but we decided against that.

Discussed options where:
1. pressing push button resets the network settings; so you can access the device again. And then in the menus have a reset to defaults feature.
2. pressing push button a few seconds resets only the network settings. Pressing it thirty settings resets all to defaults.
3. pressing push button resets all; and don't have any choices.

See image for the chose feature.

Contents of a random data partition:
```
./home/root/.bash_history
./home/root/2625428.set
./home/root/.python-history
./home/root/2046821656.rvsc
./home/root/temp.log
./home/root/.ssh
./home/root/.ssh/known_hosts
./home/root/snapshot.png
./home/root/Settings
./home/root/Settings/Trolltech.conf
./home/root/2625428.ram
./home/root/t.test
./home/vnctunnel
./home/vnctunnel/.ssh
./home/vnctunnel/.ssh/authorized_keys
./home/vnctunnel/.ssh/id_rsa.pub
./home/vnctunnel/.ssh/id_rsa
./var/lib/connman
./var/lib/connman/ethernet_028191859555_cable
./var/lib/connman/ethernet_028191859555_cable/data
./var/lib/connman/ethernet_028191859555_cable/settings
./var/lib/connman/settings
./var/lib/serial-starter/ttyUSB0
./var/lib/serial-starter/ttyUSB3
./var/lib/serial-starter/ttyUSB5
./var/lib/serial-starter/ttyUSB14
./var/lib/serial-starter/ttyUSB10
./var/lib/serial-starter/ttyUSB9
./var/lib/serial-starter/ttyUSB6
./var/lib/serial-starter/ttyUSB4
./var/lib/serial-starter/ttyUSB7
./var/lib/serial-starter/ttyUSB1
./var/lib/serial-starter/ttyUSB12
./var/lib/serial-starter/ttyUSB8
./var/lib/serial-starter/ttyS2
./var/lib/serial-starter/ttyS3
./var/lib/serial-starter/ttyUSB2
./var/lib/serial-starter/ttyUSB13
./var/lib/serial-starter/ttyS1
./var/lib/serial-starter/ttyUSB11
./var/lib/serial-starter/ttyUSB15
./var/lib/random-seed
./var/lib/mk2-dbus
./var/lib/mk2-dbus/mkxport-ttyS3.settings
./var/lib/mk2-dbus/mkxport.settings
./vrmfilescache
./log/*
./keys/ssh_host_rsa_key
./keys/ssh_host_ecdsa_key
./keys/ssh_host_ecdsa_key.pub
./keys/ssh_host_dsa_key.pub
./keys/ssh_host_rsa_key.pub
./keys/ssh_host_dsa_key
./etc/timestamp
./lost+found
./themes/overlay
./venus/installer-version
./venus/wpa-psk
./venus/unique-id
./venus/serial-number
./db/vrmlogger-backlog.sqlite3
./conf/mosquitto.d
./conf/mosquitto.d/vrm_bridge.conf
./conf/settings.xml
./conf/vrm_auth_token.txt
./conf/vncpassword.txt
```

**Comment by mpvader** [comment]
https://victrondevelopment.slack.com/archives/C9QE9QRPW/p1555421247005000

**Comment by jhofstee** [comment]
included in v2.40~36
